Code/BinkMovie/subtitleparser.cpp

```diff
@@ -273,6 +273,10 @@ bool SubTitleParserClass::Find_Movie_Entry(const char* moviename)
 
 bool SubTitleParserClass::Parse_Sub_Title(wchar_t* string, SubTitleClass* subTitle)
 {
+	// OpenW3D @fix cfehunter 07/03/2025
```

Comment:
Do we want the code filled with comments like this?
We have git version control for this.

Reply:
I don't really care either way, this is just a standard that SuperHackers is using for Generals. That said, some of these are comments where I picked them up from cfehunters fork, but I've worked them into a different commit authored by me covering different changes so the attribution will be wrong in version control.